src/server.rs

```diff
@@ -13,6 +12,9 @@ use ring::digest::{digest, SHA1_FOR_LEGACY_USE_ONLY};
 use crate::channels::{
     create_best_effort_txn_channel, create_mutated_txn_channel, create_read_only_txn_channel,
 };
+use crate::models::AlterPayload;
+
+type GenericError = Box<dyn std::error::Error + Send + Sync>;
```

Comment:
I wonder if it makes more sense to use the specific `hyper::Error` here instead.

Reply:
But this is nice

Reply:
I am using this error type on `do_alter` function with `?` syntax on lines ,151,152 and 154. I did not want to handle those errors separately, so i am letting them propagate as generic error to line 63 and handling all together